<launch>

  <!-- 
    Start Nao 2
  -->
  <arg name="nao_ip2" default="$(optenv NAO_IP 127.0.0.1)" />
  <arg name="nao_port2" default="$(optenv NAO_PORT 9559)" />
  
  <node pkg="nao_driver" type="nao_walker.py" name="nao_walker2" required="true" args="--pip=$(arg nao_ip2) --pport=$(arg nao_port2)" output="screen">
  	<remap from="cmd_vel" to="cmd_vel2"/>
  </node>
  
  <node pkg="nao_driver" type="nao_sensors.py" name="nao_sensors2" required="true" args="--pip=$(arg nao_ip2) --pport=$(arg nao_port2)" output="screen">
  	<remap from="torso_odometry" to="torso_odometry2"/>
  </node>
  
  <node pkg="nao_driver" type="nao_controller.py" name="nao_controller2" required="true" args="--pip=$(arg nao_ip2) --pport=$(arg nao_port2)" output="screen"/>
  
  
  <!--
    Nao controller 2
  -->
  <node pkg="collaborative_motion" type="nao_tracker_odometry" name="nao_tracker_odometry2" args="2" output="screen">
    <param name="NAO_IP" value="$(arg nao_ip2)"/>
    <param name="NAO_PORT" value="$(arg nao_port2)"/>
    <param name="x_01" value="1"/>
    <param name="y_01" value="-1"/>
    <param name="theta_01" value="3.14159265"/>
    <param name="x_02" value="0"/>
    <param name="y_02" value="0"/>
    <param name="theta_02" value="0"/>
  </node>

</launch>
